modules/gdscript/tests/scripts/parser/features/float_notation.out

```diff
@@ -0,0 +1,73 @@
+GDTEST_OK
+>> WARNING
+>> Line: 3
+>> ASSERT_ALWAYS_TRUE
+>> Assert statement is redundant because the expression is always true.
```

Comment:
Given this warning isn't relevant for the test, I feel it's better to replace `assert` by `print`.